specs/altair/beacon-chain.md

```diff
@@ -661,3 +662,51 @@ def process_sync_committee_updates(state: BeaconState) -> None:
         state.current_sync_committee = state.next_sync_committee
         state.next_sync_committee = get_sync_committee(state, next_epoch + EPOCHS_PER_SYNC_COMMITTEE_PERIOD)
 ```
+
+## Initialize state for pure Altair testnets and test vectors
+
+This helper function is only for initializing the state for pure Altair testnets and tests.
```

Comment:
We should note explicitly that it is for the case when `GENESIS_SLOT == ALTAIR_FORK_SLOT`

Reply:
I think it would be confusing if we add `GENESIS_SLOT == ALTAIR_FORK_SLOT` here. Two reasons:
1. For pure Altair testnets, it actually doesn't need `ALTAIR_FORK_SLOT` since we don't call `upgrate_to_altair` at all.
2. `ALTAIR_FORK_SLOT` it's inconsistent with the config files we provide. As you said, I agreed that it makes sense to set fork epoch in a separate file.